courses/factories.py

```diff
@@ -17,6 +17,7 @@ class ProgramFactory(DjangoModelFactory):
     title = fuzzy.FuzzyText(prefix="Program ")
     live = fuzzy.FuzzyAttribute(FAKE.boolean)
     description = fuzzy.FuzzyText()
+    exam_series_code = "DEDP_MicroMasters"
```

Comment:
I do not think this should have this constant: maybe a fuzzy text is better or if you really need a static text, choose something more generic.